deploy/packaging/redhat/redhat_build_rpms.py

```diff
@@ -97,15 +97,15 @@ def build():
     root = common.get_root()
     bin_packages = list()
     noarch_packages = list()
-    for package in root.getiterator('package'):
+    for package in root.iter('package'):
         attr = package.attrib
         if attr["arch"] == "noarch":
             noarch_packages.append(package)
         else:
             bin_packages.append(package)
     architectures = [{"target": "x86_64-linux",
                       "bits": 64, "arch_t": ".x86_64"}]
-    if info['dist'] != 'el8':
+    if info['dist'] != 'el8' and info['dist'] != 'el9':
```

Comment:
Shouldn't this be 'or'

Reply:
I actually did a double take on the same place

It only enables 32-bit when it's neither el8 nor el9, so `!= 8 && != 9`